akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -271,7 +350,7 @@ final class ORSet[A] private[akka] (
    * INTERNAL API
    */
   private[akka] def clear(node: UniqueAddress): ORSet[A] =
-    assignAncestor(copy(elementsMap = Map.empty))
+    assignAncestor(copy(elementsMap = Map.empty)) // FIXME delta?
```

Comment:
This should be fixed, either by (somehow) forcing full replication or by emitting list of relevant deltas.